src/main/java/com/aquarium/aquarium_backend/databaseTables/UserEffectors.java

```diff
@@ -2,17 +2,15 @@
 
 import java.util.Objects;
 
-import jakarta.persistence.Entity;
-import jakarta.persistence.GeneratedValue;
-import jakarta.persistence.Id;
-import jakarta.persistence.JoinColumn;
-import jakarta.persistence.ManyToOne;
+import jakarta.persistence.*;
```

Comment:
In my opinion, multiple imports should stay because they make code easier to read, and there is no risk of naming conflicts between two different classes

Reply:
Resolved, good one @mikic202.